docs/doc.md

```diff
@@ -8,6 +8,16 @@ The shards are run on a simple longest-chain-rule proof of stake system, where t
 
 Most users of the sharding system will run both (i) either a full (O(c) resource requirements) or light (O(log(c)) resource requirements) node on the main chain, and (ii) a "shard client" which talks to the main chain node via RPC (assumed to be trusted because it's also running on the user's computer) and can be used as a light client for any shard, as a full client for any specific shard (the user would have to specify that they are "watching" a specific shard) or as a validator node. In all cases, the storage and computation requirements for a shard client will also not exceed O(c) (unless the user chooses to specify that they are watching _every_ shard; block explorers and large exchanges may want to do this).
 
+In this document, the term `Collation` is used to differentiate from `Block` because (i) they are different RLP objects and (ii) it’s clearer in context of sharding. Basically, `Collation` must consist of `CollationHeader` and `TransactionList`; `Witness` and the detailed format of `Collation` will be defined in **Stateless clients** section. `Collator` is the collation proposer sampled by `getEligibleProposer` function of **Validator Manager Contract** in the main chain; the mechanism will be introduced in the following sections.
```

Comment:
May I ask why there is `;` between `TransactionList ` and `Witness` instead of `,`?

Reply:
Because after adjusting the titles, the 2nd level headers are: 
* Introduction
* Quadratic sharding
* Stateless clients
* Protocol changes
* Subsequent phases

`Collation` must have `Collation` and `TransactionList` in basic sharding, but `Witness` is (only) used for implementing sharding with stateless client paradigm.

Reply:
Maybe add a sentence that explains the goal of the final structure: transactions are level 0 objects, collations are level 1 objects that package transactions, and blocks are level 2 objects that package collation (headers). This was the goal of using the term collation rather than block.